src/bridge/cockpitrouter.c

```diff
@@ -375,11 +397,31 @@ create_channel (CockpitRouter *self,
                 GType type)
 {
   CockpitChannel *local;
+  const gchar *group_name;
+  ChannelGroup *group;
+
+  if (!cockpit_json_get_string (options, "group", "default", &group_name))
+    g_warning ("%s: caller specified invalid 'group' field in open message", channel);
```

Comment:
Good catch. I don't know if it's ok to just close the transport at that point. We've done it like this before and there are other places that do it like this - do you mind if this becomes a follow up?